docs/source/operations/projections/tmerc.rst

```diff
@@ -115,60 +115,211 @@ Parameters
 Mathematical definition
 #######################
 
-The formulas describing the Transverse Mercator below are quoted from Evenden's [Evenden2005]_.
+The formulation given here for the Transverse Mercator projection is due
+to Krüger :cite:`Krueger1912` who gave the series expansions accurate
+to :math:`n^4`, where :math:`n = (a-b)/(a+b)` is the third flattening.
+These series were extended to sixth order by Engsager and Poder in
+:cite:`Poder1998` and :cite:`Engsager2007`.  This gives full
+double-precision accuracy withing 3900 km of the central meridian
+:cite:`Karney2011tm`.
 
-:math:`\phi_0` is the latitude of origin that match the center of the map. It can be set with ``+lat_0``.
+This formulation consists of three steps: a conformal projection from
+the ellipsoid to a sphere, the spherical transversion Mercator
+projection, rectifying this projection to give constant scale on the
+central meridian.
 
-:math:`k_0` is the scale factor at the natural origin (on the central meridian). It can be set with ``+k_0``.
+The scale on the central meridian is :math:`k_0` and is set by ``+k_0``.
 
-:math:`M(\phi)` is the meridional distance.
+Option :option:`+lon_0` sets the central meridian; in the formulation
+below :math:`\lambda` is the longitude relative to the central meridian.
+
+Options :option:`+lat_0`, :option:`+x_0`, and :option:`+y_0` serve to
+translate the projected coordinates so that at :math:`(\phi, \lambda) =
+(\phi_0, \lambda_0)`, the projected coordinates are :math:`(x,y) =
+(x_0,y_0)`.  To simplify the formulas below, these options are set to
+zero (their default values).
 
 Spherical form
 **************
 
-Forward projection
-==================
+Because the full (ellipsoidal) projection includes the spherical
+projection as one of the components, we present the spherial form first
+with the coordinates tagged with primes, :math:`\phi'`,
+:math:`\lambda'`, :math:`\zeta'`, :math:`x'`, :math:`y'`, so that they
```

Comment:
I have some trouble to see where zeta or zeta' is actually use to compute geographic or projected coordinates